### PR TITLE
#6636 cards update

### LINKED ISSUE #6636: NFT cards imperfections + background
image
- currently, there is a tiny gap between the stroke and the image - it's basically the old stroke that is replaced with an outline when the item is in the cart.
- Can we please remake the card so there is no gap when selected

### Bg change 
- to further distinguish selected items from non-selected, we change the background. 
- cards will look like this now
image
- page in handoff updated

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #6636
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)

## Screenshot 📸

- [x] My fix has changed UI

Screen Shot 2023-08-13 at 5 21 01 PM
Screen Shot 2023-08-13 at 5 21 07 PM

### CODE REVIEW

**Review by vikiival — APPROVED** [review]

**Comment by exezbcz** [comment]
good, the gap is gone but now the stroke is inside and is making the content smaller everytime you add it to cart
- it should be outside stroke - something like outline worked, but there was a gap

File: components/items/ItemsGrid/ItemsGridImage.vue [hunk 1]
```diff
@@ -120,8 +120,8 @@ const onClickShoppingCart = () => {
 }
 .in-cart-border {
   @include ktheme() {
-    outline: 2px solid theme('k-blue') !important;
-    border-color: transparent !important;
+    border: 2px solid theme('k-blue') !important;
```
[the remark this answers is not in the document]
Comment (prachi00) [reply components/items/ItemsGrid/ItemsGridImage.vue]: fixed with outline

File: components/items/ItemsGrid/ItemsGridImage.vue [hunk 2]
```diff
@@ -121,7 +121,9 @@ const onClickShoppingCart = () => {
 .in-cart-border {
   @include ktheme() {
     outline: 2px solid theme('k-blue') !important;
+    outline-offset: -1px;
```
Comment (daiagi) [inline components/items/ItemsGrid/ItemsGridImage.vue:124]: Nice solution. Didn't know about outline offset

File: components/items/ItemsGrid/ItemsGridImage.vue [hunk 3]
```diff
@@ -121,7 +121,9 @@ const onClickShoppingCart = () => {
...
     border-color: transparent !important;
+    background-color: theme('blue-light-cards');
```
Comment (daiagi) [inline components/items/ItemsGrid/ItemsGridImage.vue:126]: This doesn't belong under `in-cart-border` clasd

Comment (prachi00) [reply components/items/ItemsGrid/ItemsGridImage.vue:126]: what do you mean?

Comment (daiagi) [reply components/items/ItemsGrid/ItemsGridImage.vue:126]: image

class named `in-cart-border` should not affect `background` properties

so, change the name of the class or separate the background into a new selector

**Comment by prachi00** [comment]
Screen Shot 2023-08-16 at 10 10 25 PM
trying to push code, I get this
@vikiival @yangwao

**Comment by vikiival** [comment]
> trying to push code, I get this

Please fork the repo under your personal profile and do a PR against upstream as usual

**Review by exezbcz — APPROVED** [review]

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $18 @ 4.77 USD/DOT ~ 3.774 $DOT
🧗 [13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1](https://polkadot.subscan.io/account/13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)
🔗 [0x73254d419db0ee0eea274e8f85de91c3e267767a0b0828f47ad28267d35531fd](https://polkadot.subscan.io/extrinsic/0x73254d419db0ee0eea274e8f85de91c3e267767a0b0828f47ad28267d35531fd)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**